packages/api/src/command/medical/document/document-query.ts

```diff
@@ -196,3 +203,15 @@ function getOrGenerateRequestId(docQueryProgress: DocumentQueryProgress | undefi
   }
   return uuidv7();
 }
+
+/**
+ * Checks if the patient's list of associated facility IDs contains the facility ID parameter passed by the user.
+ *
+ * @param patient The patient for which to check the association with a facility
+ * @param facilityId The ID of the facility passed in by the user
+ * @returns a boolean indicating whether this patient is associated with this facility ID
+ */
+function isPatientAssociatedWithFacility(patient: Patient, facilityId: string): boolean {
+  if (patient.facilityIds.includes(facilityId)) return true;
+  return false;
+}
```

Comment:
This seems like a good candidate for a domain-level, reusable function. Could move it to a file under `src/domain/medical/`, maybe `patient-facility.ts` or similar?

Super nit and mostly FYI: could replace the if/else logic w/ `.some()`:

```ts
return patient.facilityIds.some(facilityId);
```